style: lowercase graph titles (and other content changes)

jaxonL opened the pull request:
### changes
- lowercase graph titles
  - unless the word is a proper noun or acronym
  - graph labels keep their capitalisations
- content changes:
  - move cat/dog person description to misc section
  - use US instead of USA ([see here](https://english.stackexchange.com/questions/159018/the-usa-vs-the-us)]
  - some rewording of graph titles and related text
- replace male/female with men/women
- add punctuation to big spans of text (not the ones meant as a title, as in the stories section)

### other
- linechart label improvements! will now increase the svg size by that little bit that we need for it to display our labels instead of cutting it off (happened when i was testing while zoomed)

  before:
  ![image](https://user-images.githubusercontent.com/11150920/123526498-55f58100-d6a6-11eb-9418-d1dc030845e2.png)

  after:
  ![image](https://user-images.githubusercontent.com/11150920/123526478-32323b00-d6a6-11eb-8e09-80f6d8cf58eb.png)

- fix some offsets in the data (men/women labeling in salary, co-op types typo)
closes #36 .

jaxonL committed:
update readme with correct classprofile link

jaxonL committed:
fix typos

jaxonL committed:
lowercase all graph titles

- some changes to pie chart names to reflect the answers displayed
- some changes to label names to axes
- some content changes (typos, rewordings, punctuation)
- change male/female => men/women

jaxonL committed:
add punctuation to 'big' span text

- minor changes: add small disclaimer in lgbtq section
- minor rewordings

jaxonL committed:
move cat/dog section to misc

jaxonL committed:
change USA to US

see https://english.stackexchange.com/questions/159018/the-usa-vs-the-us

roxelo commented on README.md:
```diff
@@ -1,23 +1,27 @@
-The Software Engineering Class Profile is a student-initiative focused on showcasing what the program was like for students. It covers content covering co-op, background, academics, lifestyle, future plans, and answers questions such as “Does it matter if I started coding later than everyone else?” or “How much debt do students graduate with?”.
 
-You can view the class profile at [https://classprofile.andyzhang.net](https://classprofile.andyzhang.net).
+The Software Engineering Class Profile is a student-initiative focused on showcasing what the program was like for students. It covers content covering co-op, background, academics, lifestyle, future plans, and answers questions such as "Does it matter if I started coding later than everyone else?" or "How much debt do students graduate with?".
```
We should remove `""Does it matter if I started coding later than everyone else?"` as we actually don't have any graph answering this question anymore

jaxonL replied:
switched it for another question :>

roxelo commented on content/background.html:
```diff
@@ -87,7 +87,7 @@ <h3> International Students </h3>
 </p>
 <div class="half second">
-  <span class="graph-title">Proportion of international students</span>
+  <span class="graph-title">nationality breakdown</span>
```
I don't think we should use the word nationality here since the graph shows only the number of international students and domestic students. And students could have multiple nationalities

jaxonL replied:
changed to "domestic (Canadian) vs international student breakdown" in 30296c1. the reason i had changed it was because the graph displays "Canadian" and "international", which imo didn't quite match the graph title..? you're right tho, in that we only see whether a student was canadian or not (open to other suggestions of graph titles)

roxelo commented on content/background.html:
```diff
@@ -240,10 +240,10 @@ <h3>CEGEP</h3>
 </p>
 <div class="half second">
-  <span class="graph-title">Quebec Students & CEGEP</span>
+  <span class="graph-title">quebec students & CEGEP</span>
```
I think Quebec needs to be uppercased

roxelo reviewed (commented):
Small nits, also I am wondering if we should also change the graph titles to use the word `respondent` instead of `student` now since we are changing all the graph title already?

jaxonL committed:
address PR comments

- remove unasked question in readme
- cap Quebec
- rename graph to reflect data presented

jaxonL commented:
> 
> 
> Small nits, also I am wondering if we should also change the graph titles to use the word `respondent` instead of `student` now since we are changing all the graph title already?

i do think we use this interchangeably; maybe we specify this at the beginning (in the introduction?) so that people don't get the wrong idea? i do feel respondents to come off as pretty "cold" when reading graph titles

AkshayPall commented on content/coop.html:
```diff
@@ -311,23 +312,22 @@ <h3> Gender </h3>
-  Although Women are paid less than men, this is not necessarily related to a gender bias. Some
-  possible reasons could be different interests in terms of work location, type of work, etc among
+  Although women are paid less than men, this is not necessarily related to a gender bias. Some
+  possible reasons could be due to different interests in terms of work location, type of work, etc. among
```
I would rather reword this to say "it is difficult to determine what portion of the compensation difference is due to gender vias vs. student interest etc."

Currently, it sounds too dismissive of gender-based income discrimination, even though I know we make this clear in the following paragraph.

AkshayPall commented on content/coop.html:
```diff
@@ -311,23 +312,22 @@ <h3> Gender </h3>
   men and women respondents in the class.
 
-  NOTE: there were about 42-30 men and 9-8 women who responded to coop salaries
+  NOTE: there were about 42-30 men and 9-8 women who responded to coop salaries.
```
why is this 42-30 and 9-8 instead of 30-42 and 8-9?

jaxonL replied:
i'll take a look; this was the original wording, so i hadn't questioned it haha (i had assumed it was something related to the terms, but i may be wrong)

roxelo replied:
Since the graph provides percent I just wanted to clarify that the responses for each term and gender varied between 42 to 30 men and 9 to 8 women, I didn't think of the order

AkshayPall reviewed (changes requested):
Looks great! I just have the 2 comments above then let's merge!

Thank you for making it so squeaky clean 🧼✨

jaxonL committed:
reword coop gender pay section (per PR comments)

jaxonL committed:
fix linechart labels

- make svg a bit larger if labels exceed width
- properly label the men/women salary chart
- fix typo and variable name 'collision'

jaxonL committed:
fix: typo for grouped barchart leading to errors

remove barOptions (unused/outdated)
